package.json

```diff
@@ -15,7 +15,7 @@
   "author": "Qiang Xue",
   "homepage": "http://www.yiiframework.com",
   "devDependencies": {
-    "browser-sync": "^2.27.5",
+    "browser-sync": "^2.27.7",
```

Comment:
That's OK to do but isn't necessary. According to SemVer, ^2.27.5 would already update to ^2.27.*

Reply:
Ok